src/Commands/Translations/ImportLanguage.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace FOP\Console\Commands\Translations;
+
+use FOP\Console\Command;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ImportLanguage extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('fop:translations') 
+            ->setName('fop:translations:import')
+            ->setDescription('Import a translation by iso code')
+            ->setHelp('This command will import or update a translation package')
+            ->addUsage('./bin/console op:translations:import fr ( import fr translation package )')
+            ->addArgument(
+                'iso',
+                InputOption::VALUE_REQUIRED,
+                'iso of language to import ex: fr',
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $iso = $input->getArgument('iso');
+		$languagePackImporter = $this->getContainer()->get('prestashop.adapter.language.pack.importer');
+        $errors =  $languagePackImporter->import($iso);
+        if (empty($errors)) {
+            $this->io->text('lang '.$iso.' added !');
+        }
```

Comment:
If 1 is for failure.

Does the code must be 

```
if (!empty($errors)) {
    return 1;
}
$this->io->text('lang '.$iso.' added !');
return 0:
```

instead?

Reply:
```
if (!empty($errors)) {
    $this->io->text('lang '.$iso.' added !');
    return 1;
}

return 0:
```